day2/question3/solution3.go

```diff
@@ -0,0 +1,40 @@
+package main
+
+import (
+	"fmt"
+	"sync"
+)
+var wg sync.WaitGroup
+type person struct{
+	balance int
+	transactions []int
+}
+
+func update(p *person,add int){
+	p.balance+=add
```

Comment:
concurrency is not handled.

Reply:
u meant I should add a mutex.Lock() here,right?

Reply:
yeah, added a mutex that lock the sum pointer during updation